slowfast/config/defaults.py

```diff
@@ -233,6 +233,9 @@
 # Activation layer for the output head.
 _C.MODEL.HEAD_ACT = "softmax"
 
+# Activation checkpointing enabled or not to save GPU memory.
+_C.MODEL.ENABLE_AC = False
```

Comment:
Could we explicit the name as ActCheckpoint?

Reply:
of course.